src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -263,6 +266,14 @@ public void setTags(Set<Tag> tags) {
             return (tags != null) ? Optional.of(Collections.unmodifiableSet(tags)) : Optional.empty();
         }
 
+        public void setDone(Done done) {
+            this.done = done;
+        }
+
+        public Optional<Done> getDone() {
+            return Optional.ofNullable(done);
+        }
+
```

Comment:
Same as mentioned above! Since we are making unmark / mark its own command, we do not need to add it into `EditCommand.java` 😃